tools/ios/util.bzl

```diff
@@ -48,6 +48,7 @@ def swiftui_plugin(name, resources = [], deps = [], test_deps = []):
       hasUnitTests = False, 
       hasViewInspectorTests = True,
       test_host = test_host,
+      hasUITests = True,
```

Comment:
i think this should be a parameter for `swiftui_plugin` probably, since currently only one usage of `swiftui_plugin` has UI tests